Makefile

```diff
@@ -45,3 +53,32 @@ $(OBJECTS): pre
 clean:
 	rm -f $(OBJDIR)/*
 	rmdir $(OBJDIR)
+
+# Install to a destination directory. Defaults to /usr/local/, but you can
+# override it with the DESTDIR variable. Example:
+#
+# DESTDIR=~/bin make install
+install:
+	@mkdir -p $(DESTDIR)
+	$(foreach var,$(OBJECTS), install -m 0755 $(OBJDIR)/$(var) $(DESTDIR)/;)
+
+# Produce a tarball of the current commit; you can set the destination in the
+# ARCHIVEDIR variable.
+archive:
+	git archive --output=$(ARCHIVEDIR)/boulder-$(COMMIT_ID).tar.gz --prefix=boulder-$(COMMIT_ID)/ $(COMMIT_ID)
+
+# Building an RPM requires `fpm` from https://github.com/jordansissel/fpm
+# which you can install with `gem install fpm`.
+# It is recommended that maintainers use environment overrides to specify
+# Version and Epoch, such as:
+#
+# VERSION=0.1.9 EPOCH=52 MAINTAINER="$(whoami)" ARCHIVEDIR=/tmp make build rpm
+rpm:
+	fpm -s dir -t rpm --rpm-digest sha256 --name "boulder" \
+	  --license "Mozilla Public License v2.0" --vendor "ISRG" \
+		--url "https://github.com/letsencrypt/boulder" --prefix=/opt/boulder \
+		--version $(VERSION) --iteration $(COMMIT_ID) --epoch $(EPOCH) \
+		--package $(ARCHIVEDIR)/boulder-$(VERSION)-$(COMMIT_ID).x86_64.rpm \
+		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
+		--depends "libtool-ltdl" --maintainer "$(MAINTAINER)" \
+		test/boulder-config.json $(foreach var,$(OBJECTS), $(OBJDIR)/$(var))
```

Comment:
Seems nice for people to have a template to look at when Boulder complains that /etc/boulder/config.json does not exist.

Preferably, I'd write a script to go put it there, but I'm not an RPM expert, and IMO this isn't wrong behavior.